src/_pytest/logging.py

```diff
@@ -183,6 +239,12 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=DEFAULT_LOG_DATE_FORMAT,
         help="log date format as used by the logging module.",
     )
+    add_option_ini(
+        "--log-auto-indent",
+        dest="log_auto_indent",
+        default=None,
```

Comment:
Why do you set auto-indent by default to None and not to "true"?

Reply:
My preference is to set the variables that represent options to `None` if the option is not specified, rather than to a valid value, so that the default condition can be distinguished from the user explicitly choosing the default. This can make a difference when there are multiple ways to specify the option, and have it fall back if the option was not explicitly set.

This may not specifically relate to the setting I'm working on, but I find it to be a good practice overall because it may not be clear up front whether or not distinguishing _the default_ from _an explicit choice equal to the default_ may be necessary later.

Reply:
FTR I like this approach too.

Reply:
Thx for the explaination.